fast_pred/src/fastlightfm.cpp

```diff
@@ -190,15 +191,28 @@ void FastLightFM::load(string dir) {
   user_features = CSRMatrix::newInstance(cnpy::npz_load(dir + "/user-features.npz"));
   item_features = CSRMatrix::newInstance(cnpy::npz_load(dir + "/item-features.npz"));
 
+  init();
+}
+
+void FastLightFM::init() {
+
   assert(item_embeddings.shape[1] == user_embeddings.shape[1]);
   no_components = item_embeddings.shape[1];
+
+  number_of_items = item_features->rows;
```

Comment:
could we just use item_features->rows?

since number_of_items is always item_features->rows, we don't need another variable I guess, two variables for the same information could have potential out of sync problem.